pages/c/_cluster/explorer/index.vue

```diff
@@ -73,6 +135,15 @@ export default {
       cluster = await store.dispatch('management/find', { type: STEVE.CLUSTER, id });
     }
 
+    const resources = RESOURCES.map((resource) => {
+      const isExternal = Object.values(EXTERNAL).includes(resource.type);
```

Comment:
Make sure the type exists first (the store has a schema for it).  You may not have permission to get every type.

Apps will be a thing in the cluster, so the "external" thing won't be needed.

Reply:
Done, removed external and replaced apps with service in regards to your overall comment.